StandaloneMmPkg/Drivers/MmCommunicationDxe/MmCommunicationDxe.c

```diff
@@ -25,6 +25,189 @@ EFI_MM_COMMUNICATION_PROTOCOL  mMmCommunication = {
 
 MM_COMM_BUFFER             mMmCommonBuffer;
 EFI_SMM_CONTROL2_PROTOCOL  *mSmmControl2;
+EFI_SMM_ACCESS2_PROTOCOL   *mSmmAccess;
+BOOLEAN                    mSmmLocked = FALSE;
+BOOLEAN                    mEndOfDxe  = FALSE;
+
+//
+// Table of Protocol notification and GUIDed Event notifications that the Standalone Mm requires
+//
+MM_EVENT_NOTIFICATION  mMmEvents[] = {
+  //
+  // Declare protocol notification on DxeSmmReadyToLock protocols.  When this notification is established,
+  // the associated event is immediately signalled, so the notification function will be executed and the
+  // DXE SMM Ready To Lock Protocol will be found if it is already in the handle database.
+  //
+  { ProtocolNotify, TRUE,  &gEfiDxeSmmReadyToLockProtocolGuid, MmReadyToLockEventNotify,     &gEfiDxeSmmReadyToLockProtocolGuid, NULL },

```

Comment:
Better to use gEfiDxeMmReadyToLockProtocolGuid, which is consistent with the one in StandaloneMmCore

Reply:
Updated